packages/wallets/src/features/cfd/screens/Success/Success.tsx

```diff
@@ -33,10 +33,15 @@ const Success: React.FC<TSuccessProps> = ({
     const isDemo = data?.is_virtual;
     const landingCompanyName = data?.landing_company_name?.toUpperCase();
 
-    const marketTypeTitle =
-        marketType === 'all' && Object.keys(PlatformToTitleMapper).includes(platform)
-            ? PlatformToTitleMapper[platform]
-            : MarketTypeToTitleMapper[marketType];
+    let marketTypeTitle: string;
+    if (marketType && platform) {
+        marketTypeTitle =
+            marketType === 'all' && Object.keys(PlatformToTitleMapper).includes(platform)
```

Comment:
optional/preference: I'm often finding multi-line expressions with multiple operators and especially with ternary operator quite hard to read. I tend to only use ternary operator for basic, single line choices, if it spreads among multiple lines, typically its better to use plain simlpe if's.

e.g.

``` const isPlatformValid = platform && Object.keys(PlatformToTitleMapper).includes(platform);
const isMarketTypeAll = marketType === 'all';

let marketTypeTitle = 'Deriv Apps'; // default value

if (marketType && platform) {
    if (isMarketTypeAll && isPlatformValid) {
        marketTypeTitle = PlatformToTitleMapper[platform];
    } else {
        marketTypeTitle = MarketTypeToTitleMapper[marketType];
    }
}
```

Reply:
Updated.